core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -98,6 +125,11 @@ contract Liquidatable is PricelessPositionManager {
     );
     event LiquidationWithdrawn(address caller, uint256 withdrawalAmount, Status liquidationStatus);
 
+    /****************************************
+     *              MODIFIERS               *
+     ****************************************/
+
+    // Callable if the liquidation is in a state where it can be disputed.
```

Comment:
do you think we need this comment? It seems unneccessary, but if you keep it at least add a comment for `withdrawable` also